### PR TITLE
updated documentation

### PR DESCRIPTION
**Pull request by GWSzeto**
Updates the documentation of the modular contracts based on the recent refactor

---

## PR-Codex overview
The focus of this PR is to add Github button links to various ERC contracts and update metadata in the overview page.

### Detailed summary
- Added Github button links to ERC contracts: `TransferableERC20`, `TransferableERC721`, `TransferableERC1155`, `SequentialTokenIdERC1155`
- Updated metadata in the overview page with new content and images

> The following files were skipped due to too many changes: `apps/portal/src/app/contracts/sidebar.tsx`, `apps/portal/src/app/contracts/modular-contracts/module-contracts/erc-20/misc/creatorTokenERC20/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/module-contracts/erc-1155/metadata/openEditionMetadataERC1155/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/module-contracts/erc-721/metadata/openEditionMetadataERC721/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/core-contracts/erc-721/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/module-contracts/erc-20/minting/mintableERC20/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/module-contracts/erc-721/minting/mintableERC721/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/module-contracts/erc-1155/minting/mintableERC1155/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/core-contracts/erc-20/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/core-contracts/erc-1155/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/module-contracts/erc-1155/metadata/batchMetadataERC1155/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/module-contracts/erc-721/metadata/batchMetadataERC721/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/module-contracts/erc-1155/minting/claimableERC1155/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/get-started/deploy-modular-contract/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/module-contracts/erc-20/minting/claimableERC20/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/module-contracts/erc-721/minting/claimableERC721/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/get-started/create-core-contract/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/module-contracts/erc-721/misc/royaltyERC721/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/module-contracts/erc-1155/misc/royaltyERC1155/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/how-it-works/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/get-started/page.mdx`, `apps/portal/src/app/contracts/modular-contracts/get-started/create-module-contract/page.mdx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by gregfromstl — APPROVED** [review]

**Comment by joaquim-verges** [comment]
I see you moved the old things to get-started-old - can you just remove them instead? we can always get them back from git history if needed

**Comment by GWSzeto** [comment]
> I see you moved the old things to get-started-old - can you just remove them instead? we can always get them back from git history if needed

@joaquim-verges addressed

**Review by joaquim-verges — APPROVED** [review]

File: apps/portal/src/app/contracts/modular-contracts/(chapter2)/deploy-modular-contract/assets/allow-all-domains.png
[the remark this answers is not in the document]
Comment (GWSzeto) [reply apps/portal/src/app/contracts/modular-contracts/(chapter2)/deploy-modular-contract/assets/allow-all-domains.png:1]: This should be addressed

File: apps/portal/src/app/contracts/modular-contracts/(chapter2)/deploy-modular-contract/page.mdx [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+</Step>
+
+<Step title="Grab an API Key from the ThirdWeb Dashboard">
+Go to the [ThirdWeb Dashboard Settings](https://thirdweb.com/dashboard/settings/api-keys) and click on "Create API Key".
+
+<DocImage src={ createApiKey } />
+
+A dialog should pop up asking for the name of the key. Use `modular-contract-test-key`. For the sake of this tutorial, click on "Allow all domains". For production, make sure to only set domains that you trust.
+
+<DocImage src={ allowAllDomains } />
```
[the remark this answers is not in the document]
Comment (GWSzeto) [reply apps/portal/src/app/contracts/modular-contracts/(chapter2)/deploy-modular-contract/page.mdx]: This should also be addressed

**Pull request merged**